alns/weights/__init__.py

```diff
@@ -1,3 +1,8 @@
-from .SegmentedWeights import SegmentedWeights
-from .SimpleWeights import SimpleWeights
-from .WeightScheme import WeightScheme
+import warnings
+
+from alns.select import RouletteWheel as SimpleWeights
+from alns.select import SegmentedRouletteWheel as SegmentedWeights
```

Comment:
We should probably also import SelectionScheme as WeightScheme

Reply:
I don't think it's useful to include WeightScheme. I'm introducing a new selection scheme interface in the next PR, which will break WeightScheme.

SimpleWeights and SegmentedWeights will be unaffected, because the user interface stay the same, and can thus still be used while being deprecated.

Reply:
OK. I think we can get away with this and not bump to v5 already. If someone's running their own weight scheme it'll technically break things, but that's probably rare.